Makefile

```diff
@@ -192,7 +192,11 @@ openshiftci-presubmit-unittests:
 
 .PHONY: test-integration-cluster
 test-integration-cluster:
-	$(RUN_GINKGO) $(GINKGO_FLAGS) --junit-report="test-integration.xml" --label-filter="!nocluster && !podman" tests/integration
+	$(RUN_GINKGO) $(GINKGO_FLAGS) --junit-report="test-integration.xml" --label-filter="!unauth && !nocluster && !podman" tests/integration
+
+.PHONY: test-integration-cluster-unauth
+test-integration-cluster-unauth:
```

Comment:
Can we rename this target to include `openshit` in the name? This will make it clear that it is specific to OpenShift (and also why it is not run as part of the `test-integration` target), I think.